utils/kata-manager.sh

```diff
@@ -460,26 +470,55 @@ configure_containerd()
 		info "Backed up $cfg to $original"
 	}
 
+	local modified="false"
+
 	# Add the Kata Containers configuration details:
 
+	local comment_text
+	comment_text=$(printf "%s: Added by %s\n" \
+		"$(date -Iseconds)" \
+		"$script_name")
+
 	sudo grep -q "$kata_runtime_type" "$cfg" || {
 		cat <<-EOT | sudo tee -a "$cfg"
-[plugins]
-  [plugins."io.containerd.grpc.v1.cri"]
-    [plugins."io.containerd.grpc.v1.cri".containerd]
-      default_runtime_name = "${kata_runtime_name}"
-      [plugins."io.containerd.grpc.v1.cri".containerd.runtimes]
-        [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.${kata_runtime_name}]
-          runtime_type = "${kata_runtime_type}"
-EOT
-
-		info "Modified $cfg"
+		# $comment_text
+		[plugins]
+		  [plugins."io.containerd.grpc.v1.cri"]
+		    [plugins."io.containerd.grpc.v1.cri".containerd]
+		      default_runtime_name = "${kata_runtime_name}"
+		      [plugins."io.containerd.grpc.v1.cri".containerd.runtimes]
+		        [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.${kata_runtime_name}]
+		          runtime_type = "${kata_runtime_type}"
+		EOT
```

Comment:
question: should `EOT` be changed to `EOF` by the way? We have merged a PR https://github.com/kata-containers/tests/commit/90b95857090b351daf2a48f6ef20fcf63c4f18fe that do this replacement.

Reply:
**Very** good catch @liubin! Fixed! Looking at the static checker, I noticed a bug that we didn't catch at review time:

- https://github.com/kata-containers/tests/pull/4569

See also: https://github.com/kata-containers/kata-containers/pull/3865